27th_nov_5th/src/style.css

```diff
@@ -0,0 +1,128 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
```

Comment:
Comment and group the css, based on compoenents, since we using just one css for now.